src/Language/PureScript/Errors.hs

```diff
@@ -636,6 +638,9 @@ prettyPrintSingleError (PPEOptions codeColor full level showWiki) e = flip evalS
         where
         go TUnknown{} = True
         go _ = False
+    renderSimpleErrorMessage (AmbiguousTypeVariables t (Constraint nm ts _)) =
+      paras [ line $ "The inferred type " ++ (markCode (prettyPrintTypeAtom t)) ++ " has type variables which are not mentioned in the body of the type. Consider adding a type annotation."
```

Comment:
Sorry, one more 😄 :

Please put the type on its own line, `indent` it, and use `prettyPrintType` not `prettyPrintTypeAtom`. This will make it more consistent with other errors which display full types. The problem right now is that you risk it spilling over multiple lines.

Thanks!